lib/gui/app/models/store.ts

```diff
@@ -200,7 +200,7 @@ function storeReducer(
 							constraints.isDriveValid(drive, image) &&
 							!drive.isReadOnly &&
 							constraints.isDriveSizeRecommended(drive, image) &&
-							// We don't want to auto-select large drives execpt is autoSelectAllDrives is true
+							// We don't want to auto-select large drives except is autoSelectAllDrives is true
 							(!constraints.isDriveSizeLarge(drive) || shouldAutoselectAll) &&
```

Comment:
Looks like the `autoSelectAllDrives` variable might have been renamed to `shouldAutoselectAll` without the comment being updated to match? :man_shrugging:

Reply:
Seems to just be a thing:
https://github.com/balena-io/etcher/blob/863999eab96dc17b8107e49595c1613ff1c077db/lib/gui/app/models/store.ts#L177-L178